.github/workflows/terratest.yml

```diff
@@ -6,13 +6,19 @@ on:
   push:
     branches:
       - main
+
+env:
+  TERRATEST_CONFLUENT_CLOUD_SEED_KEY: ${{ secrets.TERRATEST_CONFLUENT_CLOUD_SEED_KEY }}
+  TERRATEST_CONFLUENT_CLOUD_SEED_SECRET: ${{ secrets.TERRATEST_CONFLUENT_CLOUD_SEED_SECRET }}
+  TERRATEST_CONFLUENT_CLOUD_EMAIL: ${{ secrets.TERRATEST_CONFLUENT_CLOUD_EMAIL }}
+  TERRATEST_CONFLUENT_CLOUD_PASSWORD: ${{ secrets.TERRATEST_CONFLUENT_CLOUD_PASSWORD }}
+  TERRATEST_GOOGLE_CREDENTIALS_STORAGE: ${{ secrets.TERRATEST_GOOGLE_CREDENTIALS_STORAGE }}
+  TERRATEST_GOOGLE_PROJECT_STORAGE: ${{ secrets.TERRATEST_GOOGLE_PROJECT_STORAGE }}
+
```

Comment:
This is ok but verify it really works. Sometimes these only work when specified in Jobs/steps.

Reply:
It works but I will change it to use `environment` in Github - related PR https://github.com/honestbank/terraform-code-infrastructure/pull/615

I will make new PR to new change (after I got merged the PR)